hfie/endpoint.py

```diff
@@ -84,14 +85,19 @@ def create(
     except requests.exceptions.RequestException as e:
         typer.secho(API_ERROR_MESSAGE, fg=typer.colors.RED)
         raise SystemExit(e)
-    typer.secho("Endpoint created successfully", fg=typer.colors.GREEN)
-    typer.echo(response.json())
+    typer.secho(
+        f"Endpoint {data['name']} created successfully in {data['provider']['vendor']} using {data['model']['repository']}",
```

Comment:
i think this wants to be `on aws` rather than `in aws` but not super important

Reply:
cef6c90